nixos/modules/programs/dconf.nix

```diff
@@ -1,55 +1,182 @@
 { config, lib, pkgs, ... }:
 
-with lib;
-
 let
   cfg = config.programs.dconf;
-  cfgDir = pkgs.symlinkJoin {
-    name = "dconf-system-config";
-    paths = map (x: "${x}/etc/dconf") cfg.packages;
-    postBuild = ''
-      mkdir -p $out/profile
-      mkdir -p $out/db
-    '' + (
-      concatStringsSep "\n" (
-        mapAttrsToList (
-          name: path: ''
-            ln -s ${path} $out/profile/${name}
-          ''
-        ) cfg.profiles
+
+  # Compile keyfiles to dconf DB
```

Comment:
Maybe include a mention of something like “expects `/etc/dconf/db/*.d` directory“ so that it is clearer to visualise what is happening when wrapping ones head around the code.